pages/blog/[...slug].vue

```diff
@@ -0,0 +1,545 @@
+              <RelatedArticleCard
+                v-for="relatedArticle in relatedArticles"
+                :key="relatedArticle._path"
+                :article="relatedArticle"
+                class="col-lg-6 related-article-card"
+              />
+            </div>
+          </div>
+
+          <template>
+            <div class="mt-5" id="disqus_thread"></div>
+          </template>
+
+          <noscript
+            >Please enable JavaScript to view the
+            <a href="https://disqus.com/?ref_noscript"
+              >comments powered by Disqus.</a
+            ></noscript
+          >
+        </ContentRenderer>
+      </article>
+    </div>
+    <CallToActionSection :content="article?.cta" />
+  </div>
+</template>
+
+<script setup>
+import { ref, onMounted } from 'vue'
+import ClockIcon from '../../components/icons/ClockIcon.vue'
+import TwitterIcon from '../../components/icons/twitter.vue'
+import FacebookIcon from '../../components/icons/facebook.vue'
+import LinkdinIcon from '../../components/icons/linkdin.vue'
+import CopyLinkIcon from '../../components/icons/copyLink.vue'
+import getSiteMeta from '../../utils/getSiteMeta'
+import { fetchArticle, fetchRelatedArticles } from '../../utils/getArticles'
+
+const route = useRoute().params
+
+const article = ref()
+const relatedArticles = ref()
+
+const formatDate = (date) => {
+  const options = { year: 'numeric', month: 'long', day: 'numeric' }
+  return new Date(date).toLocaleDateString('en', options)
+}
+
+const copyToClipboard = () => {
+  if (process.client) {
+    navigator.clipboard.writeText(window.location.href).then(() => {
+      // Notify user
+    })
+  }
+  googleAnalytics('custom_link')
+}
+
+const googleAnalytics = (platform) => {
+  gtag &&
+    gtag('event', 'share', {
+      method: platform,
+      content_type: 'blog',
+      item_id: article.value.title,
+    })
+}
+
+const loadDisqus = () => {
+  var disqus_config = function () {
+    this.page.url = `https://formester.com/blog/${route.slug}/`
+    this.page.identifier = `${route.slug}`
+  }
+
+  var d = document,
+    s = d.createElement('script')
+  s.src = 'https://formester.disqus.com/embed.js'
+  s.setAttribute('data-timestamp', +new Date())
+  ;(d.head || d.body).appendChild(s)
+}
+
+const encodedUrl = () => {
+  return encodeURIComponent(process.env.baseUrl + route.fullPath)
+}
+
+const meta = computed((article) => {
+  const metaData = {
+    type: 'article',
+    url: `https://formester.com/blog/${route.slug}/`,
+    title: article?.metaTitle,
+    description: article?.metaDescription,
+    mainImage: article?.coverImg
+      ? `https://formester.com/${article?.coverImg}`
+      : 'https://formester.com/formester-form-builder-background.png',
+    mainImageAlt:
+      article?.coverImgAlt ||
+      'Form builder showing drag and drop functionality',
+  }
+  return getSiteMeta(metaData)
+})
+
+useHead({
+  title: article?.metaTitle,
+  meta: [
+    { meta },
+    {
```

Comment:
meta is an array of objects, don't we need to destructure?

Reply:
This is resolved